sesman/config.h

```diff
@@ -59,6 +59,7 @@
 #define SESMAN_CFG_SEC_USR_GROUP     "TerminalServerUsers"
 #define SESMAN_CFG_SEC_ADM_GROUP     "TerminalServerAdmins"
 #define SESMAN_CFG_SEC_ALWAYSGROUPCHECK "AlwaysGroupCheck"
+#define SESMAN_CFG_RESTRICT_CLIPBOARD "RestrictClipboard"
```

Comment:
RestrictClipboard doesn't mention the direction of restriction.
I would prefer RestrictOutboundClipboard which is more clear.

The direction should be expressed in the code as well.